XIVComboPlugin/IconReplacer.cs

```diff
@@ -904,6 +916,116 @@ private ulong GetIconDetour(byte self, uint actionID)
                     return actionID;
                 }
             }
+            
+             //Pictomancer
+            if (Configuration.ComboPresets.HasFlag(CustomComboPreset.PictoSubtractivePallet))
+            {
+                if (actionID == PCT.Fire1)
+                {
+                    if (SearchBuffArray(PCT.SubPallet))
+                        return iconHook.Original(self, PCT.Bliz1);
+                    return iconHook.Original(self, PCT.Fire1);
+                }
+
+                if (actionID == PCT.Fire2)
+                {
+                    if (SearchBuffArray(PCT.SubPallet))
+                        return iconHook.Original(self, PCT.Bliz2);
+                    return iconHook.Original(self, PCT.Fire2);
+                }
+            }
+
+            if (Configuration.ComboPresets.HasFlag(CustomComboPreset.PictoHolyWhiteCombo))
+            {
+                if (actionID == PCT.HolyWhite)
+                {
+                    if (SearchBuffArray(PCT.Monochrome))
+                        return PCT.CometBlack;
+                    return PCT.HolyWhite;
+                }
+            }
+
+            if (Configuration.ComboPresets.HasFlag(CustomComboPreset.PictoMotifMuseFeature))
+            {
+                if (actionID == PCT.CreatureMotif)
+                {
+                    var PCTGauge = JobGauges.Get<PCTGauge>();
+                    if (PCTGauge.CreatureMotifDrawn)
+                        return iconHook.Original(self, PCT.LivingMuse);
+                    return iconHook.Original(self, actionID);
+                }
+
+                if (actionID == PCT.WeaponMotif)
+                {
+                    var PCTGauge = JobGauges.Get<PCTGauge>();
+                    if (PCTGauge.WeaponMotifDrawn)
+                        return iconHook.Original(self, PCT.SteelMuse);
+                    if(SearchBuffArray(PCT.HammerReady))
+                        return iconHook.Original(self, PCT.HammerStamp);
+                    return iconHook.Original(self, actionID);
+                }
+            }
+
+            if (Configuration.ComboPresets.HasFlag(CustomComboPreset.PictoStarrySkyCombo))
+            {
+                if (actionID == PCT.LandscapeMotif)
+                {
+                    var PCTGauge = JobGauges.Get<PCTGauge>();
+                    if (PCTGauge.LandscapeMotifDrawn)
+                        return PCT.StarryMuse;
+                    if (SearchBuffArray(PCT.StarStruck))
+                        return PCT.StarPrism;
+                    return PCT.StarryMotif;
+                }
+            }
```

Comment:
Here's the implementation for my above comment. Playing with it feels pretty good and should let people customize it how they want to play
```suggestion
            if (Configuration.ComboPresets.HasFlag(CustomComboPreset.PictoMotifMuseFeature))
            {
                if (actionID == PCT.CreatureMotif)
                {
                    var PCTGauge = JobGauges.Get<PCTGauge>();
                    if (PCTGauge.CreatureMotifDrawn){
                        if (Configuration.ComboPresets.HasFlag(CustomComboPreset.PictoLivingMuseCombo))
                        {
                            if (PCTGauge.MooglePortraitReady || PCTGauge.MadeenPortraitReady)
                                return iconHook.Original(self, PCT.MogOfTheAges);
                        }
                        return iconHook.Original(self, PCT.LivingMuse);
                    }

                    return iconHook.Original(self, actionID);
                }

                if (actionID == PCT.WeaponMotif)
                {
                    var PCTGauge = JobGauges.Get<PCTGauge>();
                    if (PCTGauge.WeaponMotifDrawn)
                    {
                        if (Configuration.ComboPresets.HasFlag(CustomComboPreset.PictoSteelMuseCombo))
                        { 
                            if (SearchBuffArray(PCT.HammerReady))
                                return iconHook.Original(self, PCT.HammerStamp); 
                        }
                        return iconHook.Original(self, PCT.SteelMuse);
                    }
                    return iconHook.Original(self, actionID);
                }

                if (actionID == PCT.LandscapeMotif)
                {
                    var PCTGauge = JobGauges.Get<PCTGauge>();
                    if (PCTGauge.LandscapeMotifDrawn){
                        if (Configuration.ComboPresets.HasFlag(CustomComboPreset.PictoScenicMuseCombo))
                        {
                            if (SearchBuffArray(PCT.StarStruck))
                                return iconHook.Original(self, PCT.StarPrism);
                        }
                        return iconHook.Original(self, PCT.ScenicMuse);
                    }
                    return iconHook.Original(self, PCT.StarryMotif);
                }
            } else
            {
                if (Configuration.ComboPresets.HasFlag(CustomComboPreset.PictoLivingMuseCombo))
                {
                    if (actionID == PCT.LivingMuse)
                    {
                        var PCTGauge = JobGauges.Get<PCTGauge>();
                        if (Configuration.ComboPresets.HasFlag(CustomComboPreset.PictoLivingMuseCombo))
                        {
                            if (PCTGauge.MooglePortraitReady || PCTGauge.MadeenPortraitReady)
                                return iconHook.Original(self, PCT.MogOfTheAges);
                        }
                        return iconHook.Original(self, actionID);
                    }
                }

                if (Configuration.ComboPresets.HasFlag(CustomComboPreset.PictoSteelMuseCombo))
                {
                    if (actionID == PCT.SteelMuse)
                    {
                        if (SearchBuffArray(PCT.HammerReady))
                            return iconHook.Original(self, PCT.HammerStamp);
                        return iconHook.Original(self, actionID);
                    }
                }

                if (Configuration.ComboPresets.HasFlag(CustomComboPreset.PictoScenicMuseCombo))
                {
                    if (actionID == PCT.ScenicMuse)
                    {
                        if (SearchBuffArray(PCT.StarStruck))
                            return iconHook.Original(self, PCT.StarPrism);
                        return iconHook.Original(self, actionID);
                    }
                }
            }
```

Reply:
1. I think it's unnecessary to break out Living/Steel Muse into separate combos. If you're doing one, you're probably doing the other. But Starry Muse is a little bit trickier because you have to check for the follow-up buff.
2. This duplicates a lot of code onto the Muse button. I see no reason to alter the Muse button if there's a straightforward way to put everything onto the Motif button.

Reply:
1. In playing with it, I found I don't like having `Mog of the Ages` replace `Living Muse` because sometimes you end up in weird situations where you have a creature motif and the moogle portrait at the same time and the cooldowns are obscured.
2. I said it would make the logic ugly 😆 In laying out my hotbars, it was more convenient to have things combo off of muse instead of motif.

Reply:
Good points. I've played around with it a little bit more and I see what you mean. Since the Creature pallet has multiple steps, you might want to preload a step while still holding onto Mog/Maween. But for Weapon/Landscape, since there aren't multiple steps you can preload, there's no reason to do it for them. So what I'm thinking is: A toggle for all Muses to go over their respective Motifs, and another for Steel/Scenic Muse follow-ups to also go on their Motif.